src/decoders/arw.rs

```diff
@@ -73,7 +73,7 @@ impl<'a> Decoder for ArwDecoder<'a> {
 }
 
 impl<'a> ArwDecoder<'a> {
-  fn image_a100(&self, camera: &Camera) -> Result<RawImage,String> {
```

Comment:
ok_image right. All the values ok_image uses should be copyable or at least clonable so there should be no problem with just taking a reference and copying what you need and then discarding.